<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plays</class>
 <widget class="QFrame" name="Plays">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>402</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="PlaysLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabSpacing">
         <number>6</number>
        </property>
        <widget class="QWidget" name="widget_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>625</width>
           <height>282</height>
          </rect>
         </property>
         <attribute name="icon">
          <iconset>
           <normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</iconset>
         </attribute>
         <attribute name="label">
          <string>Today</string>
         </attribute>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <stylestrategy>PreferDefault</stylestrategy>
             </font>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>623</width>
               <height>280</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <stylestrategy>NoAntialias</stylestrategy>
              </font>
             </property>
             <layout class="QHBoxLayout" name="today_plays_wrapper">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>625</width>
           <height>282</height>
          </rect>
         </property>
         <attribute name="icon">
          <iconset>
           <normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</iconset>
         </attribute>
         <attribute name="label">
          <string>Tomorrow</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="widget_7">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>623</width>
               <height>280</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="tomorow_plays_wrapper">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>625</width>
           <height>282</height>
          </rect>
         </property>
         <attribute name="icon">
          <iconset>
           <normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</iconset>
         </attribute>
         <attribute name="label">
          <string>this weekend</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="widget_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>623</width>
               <height>280</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="thisweekend_plays_wrapper">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>625</width>
           <height>282</height>
          </rect>
         </property>
         <attribute name="icon">
          <iconset>
           <normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</normaloff>C:/Users/admin/Desktop/img/icons/theatre-masks.png</iconset>
         </attribute>
         <attribute name="label">
          <string>next weekend</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_5">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="widget_8">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>623</width>
               <height>280</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="nextweekend_plays_wrapper">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
